core/src/main/scala/kafka/admin/ReassignPartitionsCommand.scala

```diff
@@ -194,33 +192,18 @@ object ReassignPartitionsCommand extends Logging {
 
   def main(args: Array[String]): Unit = {
     val opts = validateAndParseArgs(args)
-    var toClose: Option[AutoCloseable] = None
     var failed = true
 
+    val props = if (opts.options.has(opts.commandConfigOpt))
+      Utils.loadProps(opts.options.valueOf(opts.commandConfigOpt))
```

Comment:
Good question. The lines I moved out of `try` block are props setup (which is safe), and `Admin.create`. I was trying to do try-with-resource in scala, but it can't. I moved the `Admin.create` back to try block. Thank you.

Reply:
Why is props setup safe? It could fail to load the fail, etc.